src/pages/Judging/Done.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+export default () => (
+  <h1>
+    Done!
+  </h1>
+)
```

Comment:
we should probably add // TODO: impl or something of the like for the stub pages